api/src/convert/input_ring.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Convert to/from mc_transaction_core::ring_ct::InputRing.
+
+use crate::{external, ConversionError};
+use mc_crypto_ring_signature_signer::SignableInputRing;
+use mc_transaction_core::ring_ct::{InputRing, PresignedInputRing};
+
+impl From<&InputRing> for external::InputRing {
```

Comment:
I'm seeing `InputRing` checked for tests but I don't see the tests...

Reply:
Oh oops forgot to commit those changes